qa/rpc-tests/zkey_import_export.py

```diff
@@ -94,11 +90,11 @@ def get_private_balance(node):
         # verify_utxos(charlie, [])
 
         # the amounts of each txn embodied which generates a single UTXO:
-        amounts = map(Decimal, ['2.3', '3.7', '0.1', '0.5', '1.0', '0.19'])
+        amounts = list(map(Decimal, ['2.3', '3.7', '0.1', '0.5', '1.0', '0.19']))
```

Comment:
Personal opinion:

`amounts = [Decimal(x) for x in ['2.3', '3.7', '0.1', '0.5', '1.0', '0.19']]`

is more readable.

Reply:
In particular the use of `list` to force the lazy generator to produce a list is a bit opaque.